src/ui/qgstextformatwidgetbase.ui

```diff
@@ -5558,6 +5558,33 @@ font-style: italic;</string>
                              </property>
                             </widget>
                            </item>
+                           <item>
+                            <spacer name="horizontalSpacer_6">
```

Comment:
Looking at the screenshot this layout has resulted in a very wide horizontal minimum size on the widget. Can you instead move the new Point label and button to a new row below the x/y buttons?